test/integration/targets/eos_acls/tasks/cli.yaml

```
@@ -0,0 +1,17 @@
+---
+- name: collect all cli test cases
+  find:
+    paths: "{{ role_path }}/tests/cli"
```

```suggestion
    paths: "{{ role_path }}/tests/common"
```

I realize this won't work _now_, but with #66871 coming soon, can we move these tests to common and run them for both connections, rather than running no tests for eapi?

---

Done!